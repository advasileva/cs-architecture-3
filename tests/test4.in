gFisQJl)M:+r{cF|BTxhg
vZ.c3If2
71X#Z%]Xi^T+2MF4TtA!=\pi>!Aa|g>~j\k8\*l0;{2/<Tl[\~C