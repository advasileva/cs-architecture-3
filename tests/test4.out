21807.298273